controller/Review.php

```diff
@@ -100,68 +104,88 @@ public function init(): void
 
         $params = $this->getPsrRequest()->getQueryParams();
 
-        if (isset($params['serviceCallId'])) {
-
-            /** @var DeliveryExecutionFinderService $finder */
-            $finder = $this->getServiceLocator()->get(DeliveryExecutionFinderService::SERVICE_ID);
-
-            $data = $dataBuilder->create()->build($params['serviceCallId'], $finder->getShowScoreOption($this->ltiSession->getLaunchData()));
+        try {
+            if (isset($params['serviceCallId'])) {
+                /** @var DeliveryExecutionFinderService $finder */
+                $finder = $this->getServiceLocator()->get(DeliveryExecutionFinderService::SERVICE_ID);
```

Comment:
can we extract this so we are able to mock / change it easier?

Reply:
this service already covered by tests. Not sure that we should mock it here, in the controller